examples/otlp-exporter-node/README.md

```diff
@@ -1,10 +1,9 @@
 # Overview
 
-<!-- TODO: This example needs an update. -->
-
-This example shows how to use [@opentelemetry/exporter-otlp-http](https://github.com/open-telemetry/opentelemetry-js/tree/v0.26.0/experimental/packages/opentelemetry-exporter-otlp-http) to instrument a simple Node.js application.
-
-This example will export spans data simultaneously using [Exporter Collector](https://github.com/open-telemetry/opentelemetry-js/tree/v0.26.0/experimental/packages/opentelemetry-exporter-otlp-http) and grpc. It will use [proto format](https://github.com/open-telemetry/opentelemetry-proto).
+This example shows how to use
+[@opentelemetry/exporter-trace-otlp-http](https://github.com/open-telemetry/opentelemetry-js/tree/v0.26.0/experimental/packages/opentelemetry-exporter-trace-otlp-http)
+and [@opentelemetry/exporter-metrics-otlp-http](https://github.com/open-telemetry/opentelemetry-js/tree/v0.26.0/experimental/packages/opentelemetry-exporter-metrics-otlp-http)
```

Comment:
nit: these links point to an old version (0.26.0) and I also think the package name changed from "opentelemetry-exporter-trace-otlp-http" to "exporter-trace-otlp-http".
Might be worth fixing while we're at it.

Reply:
Yep, that's a good idea. Fixed the links in [00e7015](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/00e70155930c7bbe4ddb78a47f6969d71d862f1c). Thanks! :slightly_smiling_face: